src/main/java/it/gov/pagopa/pu/debtpositions/service/installmentsync/operation/InstallmentSynchronizeCancelServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package it.gov.pagopa.pu.debtpositions.service.installmentsync.operation;
+
+import it.gov.pagopa.pu.debtpositions.dto.generated.*;
+import it.gov.pagopa.pu.debtpositions.exception.custom.ConflictErrorException;
+import it.gov.pagopa.pu.debtpositions.service.update.DebtPositionCancelInstallmentService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Set;
+
+@Service
+@Slf4j
+public class InstallmentSynchronizeCancelServiceImpl {
+
+  private final DebtPositionCancelInstallmentService debtPositionCancelInstallmentService;
+
+  private static final Set<InstallmentStatus> installmentStatusesValidForCancel =
+    Set.of(InstallmentStatus.UNPAID, InstallmentStatus.EXPIRED, InstallmentStatus.DRAFT);
+
+  public InstallmentSynchronizeCancelServiceImpl(DebtPositionCancelInstallmentService debtPositionCancelInstallmentService) {
+    this.debtPositionCancelInstallmentService = debtPositionCancelInstallmentService;
+  }
+
+  public String syncInstallment(InstallmentSynchronizeDTO installmentSynchronizeDTO, DebtPositionDTO debtPositionDTO,
+                                Boolean massive, String accessToken, String operatorExternalUserId) {
+    if (debtPositionDTO == null) {
+      throw new ConflictErrorException(String.format("The debt position related to iupd %s was not found", installmentSynchronizeDTO.getIupdOrg()));
```

Comment:
why Conflict and not NotFound? (also when searching for the installment)?

I expect Conflict id the status is not allowed